source/documentation/using_the_api/register_components.md

```diff
@@ -1,18 +1,34 @@
-## The relationship between entries, items and records
+## The components of a register
 
-Registers are made up of:
+Registers are primarily composed of:
 
 * entries
 * items
 * records
 
+### Entries
+
 Entries contain metadata about what and when data has changed. 
 
-Each entry in a register has a relationship with one item, which contains data. An item is connected to an entry with an item hash.   
+Entries contain the fields `entry-number`,  `entry-timestamp`, `item-hash` and `key`. 
+
+The `entry-number` is unique and defines an entry’s position within the ordered list of a register. The `entry-timestamp` is the time when an entry was introduced to a given register, but it is no guarantee of the order of entries in a register. The `entry-number` is.
+
+Entries are connected to items by a given `item-hash`. 
```

Comment:
Being picky: Entries reference items in the `item-hash` field using the hash derived from the item content.

What I think it's important is to not confuse "`item-hash`" and item hash. The former is a field name, the latter is the actual alphanumeric value.

Reply:
> Each entry is connected to an item by a given item hash. Entries reference items in the item-hash field using the hash derived from the corresponding item's content.

The above looks much better, thanks.

> [...] so correct me if I'm wrong but it's not just the item hash/item-hash that has that problem in that whole part of the text but rather it's all of them (e.g. entry number, entry timestamp).

Well, in the other cases is an abuse of language but seems clear that you talk about the value of the field, not the field name itself.

Reply:
> Well, in the other cases is an abuse of language but seems clear that you talk about the value of the field, not the field name itself.

OK, so I'll correct the other cases at the same time. I think in doing that we solve both problems at the same time, but the main problem is with this sentence:

> Entries are connected to items by a given item hash. 

Which I think we can fix with: 

> Entries are connected to items by a given item hash.